test/connectors/httpParsers/SaveSubmittedProcessHttpParserSpec.scala

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package connectors.httpParsers
+
+import java.util.UUID.randomUUID
+
+import base.BaseSpec
+import connectors.httpParsers.SaveSubmittedProcessHttpParser.saveSubmittedProcessHttpReads
+import models.errors.{InternalServerError, InvalidProcessError}
+import models.{RequestOutcome, SaveSubmittedProcessResponse}
+import play.api.http.{HttpVerbs, Status}
+import play.api.libs.json.{JsValue, Json}
+import uk.gov.hmrc.http.HttpResponse
+
+class SaveSubmittedProcessHttpParserSpec extends BaseSpec with HttpVerbs with Status {
+
+  private trait Test {
+
+    val url: String = "/test"
+
+    val id: String = randomUUID().toString
```

Comment:
I wouldn't use a UUID here it could confuse people. Only scratch processes use an UUID as identifiers. It would be better to use an Ocelot identifier, such as ext90002, although ultimately the id will probably be some composite of the Ocelot id and the lastUpdate or submission date and time.

Reply:
have updated the test